Formula/vlmcsd.rb

```diff
@@ -0,0 +1,42 @@
+class Vlmcsd < Formula
+  desc "KMS Emulator in C"
+  homepage "https://github.com/Wind4/vlmcsd"
+  url "https://github.com/Wind4/vlmcsd/archive/svn1113.tar.gz"
+  version "svn1113"
+  sha256 "62f55c48f5de1249c2348ab6b96dabbe7e38899230954b0c8774efb01d9c42cc"
+  head "https://github.com/Wind4/vlmcsd.git"
+
+  depends_on "make" => :build
+  uses_from_macos "llvm" => :build
+
+  def install
+    system "make", "CC=clang"
+    bin.install "bin/vlmcsd"
+    bin.install "bin/vlmcs"
+    man1.install "man/vlmcs.1"
+    man7.install "man/vlmcsd.7"
+    man8.install "man/vlmcsd.8"
+    man5.install "man/vlmcsd.ini.5"
+  end
+
+  test do
+    # Version checks
+    output = shell_output("#{bin}/vlmcsd -V")
+    assert_match /vlmcsd/, output
+    output = shell_output("#{bin}/vlmcs -V")
+    assert_match /vlmcs/, output
+    # Test for running
+    begin
+      server = IO.popen("#{bin}/vlmcsd -D")
```

Comment:
Pleas use fork like it is here: https://github.com/Homebrew/homebrew-core/blob/master/Formula/httpd.rb#L169-L172

Reply:
It has been modified